PWGEM/PhotonMeson/Core/CutsLibrary.cxx

```diff
@@ -166,8 +166,8 @@ V0PhotonCut* o2::aod::pcmcuts::GetCut(const char* cutName)
   }
   if (!nameStr.compare("nopid")) {
     // for track
-    cut->SetTrackPtRange(0.01f, 1e10f);
-    cut->SetTrackEtaRange(-0.9, +0.9);
+    cut->SetTrackPtRange(1.01f, 1e10f);
```

Comment:
is this not supposed to be `cut->SetTrackPtRange(0.02f, 1e10f);`

Reply:
Yes. thank you.